docs/pages/data-fetching/+Page.mdx

```diff
@@ -43,19 +49,20 @@ async function data(pageContext) {
 
 > The `@id` in the file path `/pages/movie/@id/+data.js` denotes a route parameter which value is available at <Link href="/pageContext#routeParams">`pageContext.routeParams.id`</Link>, see <Link href="/routing" />.
 
-```js
-// SomeComponent.js
+```ts
+// SomeComponent.{tsx|vue}
```

Comment:
```suggestion
// SomeComponent.ts
```

I do like your idea :+1: but let's do it later and only after we have a `JSX <-> Vue` toggle.